rasa_core/server.py

```diff
@@ -407,6 +407,9 @@ def predict(sender_id):
         try:
             # Fetches the appropriate bot response in a json format
             responses = agent.predict_next(sender_id)
+            responses['scores'] = sorted(responses['scores'],
```

Comment:
would you mind adding a test for this as well? (also would be great if you can update the api documentation as well as the CHANGELOG.rst)

Reply:
have added a test and updated the api documentation and CHANGELOG.rst

Reply:
@tmbo hey, can you please review the additions?